freqtrade/freqtradebot.py

```diff
@@ -2017,6 +2065,20 @@ def execute_trade_exit(
             logger.info(f"User denied exit for {trade.pair}.")
             return False
 
+        if trade.has_open_orders:
+            if self.handle_similar_open_order(trade, limit, amount, trade.exit_side):
+                logger.info(
+                    f"A similar open order was found for {trade.pair}. \
+                    Keeping existing {trade.exit_side} order. price: {limit}, amount: {amount}"
+                )
+                return False
+
+            # cancel any open order of this trade
+            self.cancel_open_orders_of_trade(
```

Comment:
This call happens within `handle_similar_open_order()`, too ...
so i think we can safely omit this (and the subsequent commit)?

---

The alternative (and more likely scenario as to why this is here) is that this call needs "trade.entry_side" as parameter (too?) - as currently, open entry orders will not be canceled when we're exiting the trade - potentially leaving "ghost" orders around (orders freqtrade no longer checks on)